### DISCUSSION TITLE
[ERROR] SMTP error: response error: error Tag at

### DISCUSSION THREAD

**Discussion by vitaaaaa1** [dani-garcia/vaultwarden#4994] [Q&A]
### Vaultwarden Support String

[ERROR] SMTP error: response error: error Tag at

### Vaultwarden Build Version

latest

### Deployment method

Other method

### Custom deployment method

use docker compoe to make this application
this is my docker compose yaml 's file

```version: '3'
services:
  vaultwarden:
    image: vaultwarden/server:latest
    container_name: vaultwarden
    restart: always
    ports:
      - 80:80
    env_file:
      - ./vaultwarden.env
    volumes:
      - /data/vaultwarden_data:/data/
 ```

      and this is my env's file

```      
DOMAIN: https://xxxxx
ADMIN_TOKEN: xxx
SMTP_HOST: xxxx
SMTP_FROM: xxx
SMTP_PORT: 587
SMTP_SECURITY: starttls
SMTP_USERNAME: xxxxx
SMTP_PASSWORD: xxxxx
```

### Reverse Proxy

nginx

### Host/Server Operating System

Linux

### Operating System Version

docker compose

### Clients

Web Vault

### Steps To Reproduce

1.create New organization and inite member
2. and this member use email to Accept the invitation 
3. admin user need to confim this user to the New organization
4. error
![eroor](https://github.com/user-attachments/assets/b1651440-cf98-42be-a120-29433edb78e9)
5,but i can get a email, I tried to log in, but it didn't work

### Expected Result

error

### Actual Result

error

### Logs

```text
vaultwarden  | [2024-09-26 10:09:11.572][request][INFO] GET /api/users/416a7085-5be3-4693-9679-c39dba140bf7/public-key
vaultwarden  | [2024-09-26 10:09:11.574][response][INFO] (get_public_keys) GET /api/users/<uuid>/public-key => 200 OK
vaultwarden  | [2024-09-26 10:09:12.863][request][INFO] POST /api/organizations/6e7ec168-bf37-42b3-b9d7-339b5dafe429/users/b8dbf5a0-c1e2-4254-87ee-588f146b6a4e/confirm
vaultwarden  | [2024-09-26 10:09:13.967][vaultwarden::mail][ERROR] SMTP error: response error: error Tag at: 
vaultwarden  | 
vaultwarden  | [2024-09-26 10:09:13.969][response][INFO] (confirm_invite) POST /api/organizations/<org_id>/users/<org_user_id>/confirm => 400 Bad Request
```

### Additional Context

1.invite member，l can use this email to login and this user
The user will also change from the invited status bar to the Needsconfimation status bar
![image](https://github.com/user-attachments/assets/2642a202-977a-4bc0-8153-dc5452acf71d)
![image](https://github.com/user-attachments/assets/e3741fad-8c32-4d61-8a4e-0304d0ad4903)

2. when i want to confim ，it not working
![image](https://github.com/user-attachments/assets/5b47c67a-d258-480b-9bd1-2124674c313d)

**Comment by BlackDex** [comment]
Seems to be something with your SMTP Provider which returns errors.
Looks similar too #4992

**Reply by vitaaaaa1** [reply]
I used the smtp services of Tencent and NetEase in the test, they are not working properly, is there a recommended smtp provider

**Reply by stefan0xC** [reply]
Cf. https://github.com/dani-garcia/vaultwarden/wiki/SMTP-Configuration#smtp-servers

**Reply by BlackDex** [reply]
I do wonder, if all those fail what is so special or different then?
Did you translated the emails? Maybe it's the name of the user or organization? Could you provide those maybe or something similar in your language?

**Reply by vitaaaaa1** [reply]
Today I tried to use gmail and he succeeded, which confused me
